Description: Don't consider the "data" when installing a package - this is usually "Ubuntu" and isn't useful to pass to apt.
Author: Justin McPherson <justin.mcpherson@canonical.com>
Forwarded: https://code.launchpad.net/~justinmcp/aptdaemon/1352654/+merge/243354
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/aptdaemon/+bug/1352654

Index: b/aptdaemon/pkcompat.py
===================================================================
--- a/aptdaemon/pkcompat.py
+++ b/aptdaemon/pkcompat.py
@@ -1904,8 +1904,6 @@
             id += ":%s" % arch
         if version:
             id += "=%s" % version
-        if data and data not in ["local", "installed"]:
-            id += "/%s" % data
         return id
 
     def _get_merged_trans(self, role, pkg_ids=None, pkg_type=None,
